src/Morphic-Widgets-Basic/SimpleButtonMorph.class.st

```diff
@@ -48,7 +48,9 @@ Class {
 { #category : #'instance creation' }
 SimpleButtonMorph class >> newWithLabel: labelString [
 
-	^ self new label:  labelString
+	^(self new) 
```

Comment:
There is no need for parenthesis

Reply:
Yes - unnecessary but makes a small difference for the reader. 

With the braces the method is more readable as one can directly distinquish the message sent to the class from the message sent to the resulting instance. And that's why I added my comment if you look in the first commit message that this time I used the based.

It is a best practice pattern (BPP) one can but must not follow. It was once recommended either in Smalltalk with Style or in one of Kent's best practices book if I remember correctly.

More or less a matter of personal taste.